### PR TITLE
[Trino] Table/database names starting with digit should inclosed with double quotes

### PR DESCRIPTION
**Pull request by agl29**
## What changes were proposed in this pull request?

- Table/database names starting with digit were giving syntax error.
- Added the double quotes for catalog, database and tables

## How was this patch tested?

- Locally

### CODE REVIEW

**Review by Harshg999 — APPROVED** [review]
LGTM!

File: desktop/libs/notebook/src/notebook/connectors/trino.py [hunk 1]
```diff
@@ -90,21 +87,24 @@ def get_auth_password(self):
         else DEFAULT_AUTH_PASSWORD.get()
     )
 
+  def _format_database(self, database):
+    parts = database.split('.')
+    database = '"' + '"."'.join(parts) + '"'
```
Comment (JohanAhlen) [inline desktop/libs/notebook/src/notebook/connectors/trino.py:92]: Shouldn't we make sure only formats it once? I.e. not if it's already formatted..

File: desktop/libs/notebook/src/notebook/connectors/trino.py [hunk 2]
```diff
@@ -248,15 +245,15 @@ def get_sample_data(self, snippet, database=None, table=None, column=None, is_as
...
     return response
 
-
   def _get_select_query(self, database, table, column=None, operation=None, limit=100):
     if operation == 'hello':
       statement = "SELECT 'Hello World!'"
     else:
-      column = '%(column)s' % {'column': column} if column else '*'
+      database = self._format_database(database)
+      column = '"%(column)s"' % {'column': column} if column else '*'
```
Comment (JohanAhlen) [inline desktop/libs/notebook/src/notebook/connectors/trino.py:253]: Same.

File: desktop/libs/notebook/src/notebook/connectors/trino.py [hunk 3]
```diff
@@ -248,15 +245,15 @@ def get_sample_data(self, snippet, database=None, table=None, column=None, is_as
...
       statement = textwrap.dedent('''\
           SELECT %(column)s
-          FROM %(database)s.%(table)s
+          FROM %(database)s."%(table)s"
```
Comment (JohanAhlen) [inline desktop/libs/notebook/src/notebook/connectors/trino.py:256]: And here.

**Review by JohanAhlen — APPROVED** [review]

File: desktop/libs/notebook/src/notebook/connectors/trino.py [hunk 4]
```diff
@@ -15,29 +15,27 @@
...
 # limitations under the License.
 
-import logging
-import json
-import requests
 import sys
-import textwrap
+import json
 import time
+import logging
```
Comment (wing2fly) [inline desktop/libs/notebook/src/notebook/connectors/trino.py:21]: This seems not using anywhere in the file

File: desktop/libs/notebook/src/notebook/connectors/trino.py [hunk 5]
```diff
@@ -47,7 +45,7 @@ def decorator(*args, **kwargs):
     except RestException as e:
       try:
         message = force_unicode(json.loads(e.message)['errors'])
-      except:
+      except Exception as e:
```
Comment (wing2fly) [inline desktop/libs/notebook/src/notebook/connectors/trino.py:48]: Should this change the character to not shadowed by the previous exception, like `Exception as ex`?

File: desktop/libs/notebook/src/notebook/connectors/trino.py (hunk 1 above)
Comment (agl29) [reply desktop/libs/notebook/src/notebook/connectors/trino.py:99]: Done

File: desktop/libs/notebook/src/notebook/connectors/trino.py (hunk 3 above)
Comment (agl29) [reply desktop/libs/notebook/src/notebook/connectors/trino.py]: condition added

File: desktop/libs/notebook/src/notebook/connectors/trino.py (hunk 2 above)
Comment (agl29) [reply desktop/libs/notebook/src/notebook/connectors/trino.py]: done

File: desktop/libs/notebook/src/notebook/connectors/trino.py (hunk 5 above)
Comment (agl29) [reply desktop/libs/notebook/src/notebook/connectors/trino.py]: done

**Review by Harshg999 — APPROVED** [review]

**Pull request merged**